lib/setup.py

```diff
@@ -79,7 +79,12 @@
 if not os.getenv("SNOWPARK_CONDA_BUILD"):
     INSTALL_REQUIRES.extend(SNOWPARK_CONDA_EXCLUDED_DEPENDENCIES)
 
-EXTRA_REQUIRES = {"snowflake": ["snowflake-snowpark-python; python_version=='3.8'"]}
+EXTRA_REQUIRES = {
+    "snowflake": [
+        "snowflake-snowpark-python; python_version=='3.8'",
```

Comment:
Any chance we can specify the right version here? It needs to be at least `0.9.0` to make the tests pass. I'm sure this download the latest version, but we _expect_ this version of snowflake-snowpark-python,